src/test/java/com/cloudbees/jenkins/support/impl/SmartLogCleanerTest.java

```diff
@@ -33,20 +31,19 @@
 
     @Test
     public void cleanUp() throws Exception {
-
         File supportDir = new File(j.getInstance().getRootDir(), "support");
         File cacheDir = new File(supportDir, "winsw");
 
         DumbSlave slave1 = j.createOnlineSlave();
         DumbSlave slave2 = j.createOnlineSlave();
         generateBundle();
 
-        assertNotNull("The cache directory is empty",cacheDir.list());
+        assertNotNull("The cache directory is empty", cacheDir.list());
 
         // wait for completion of SmartLogFetcher async tasks during the bundle generation
-        for (int i = 0; i <  10; i++) {
+        for (int i = 0; i < 10; i++) {
             int cacheDirsCount = cacheDir.list().length;
-            if (cacheDir != null && cacheDir.list() != null && cacheDirsCount == 2) {
+            if (cacheDir.list() != null && cacheDirsCount == 2) {
```

Comment:
You could actually skip the nullability check as well given the previous assertion?

Reply:
done